src/content_scripts/wtm-report/index.js

```diff
@@ -57,6 +57,12 @@ function renderWheel(anchor, stats) {
   const parent = anchor.parentElement;
   parent.style.position = 'relative';
   const threeDotsElement = parent.querySelector('div[jsslot] div[aria-haspopup], div[jsaction] div[role=button] span');
+
+  if (!threeDotsElement) {
```

Comment:
Perhaps the matching code for Chrome Desktop works on mobile? If not, maybe we should skip it only for Mobile. Otherwise, the rendering will effectively be disabled on non-Safari browsers.

Reply:
I've updated the `isMobile` check, it should cover all mobile devices now